pages/settings/account.tsx

```diff
@@ -0,0 +1,21 @@
+import SettingsBarItem from "@/Components/SettingsBar/Item";
+import { HomeSidebar } from "@/Components/components";
```

Comment:
Why named import? We should use default import.

```suggestion
import HomeSidebar from "@/Components/components/HomeSidebar";
```

Reply:
Yes, Sir, this seems to be more clearer than my code. Thank you for your valuable suggestion. 👍😊

Reply:
https://github.com/PiyushKalyanpy/GyanaGuru/pull/264/commits/9c04240d2bdd8d56ee4b38f834580ccc4a22b328